src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -29,28 +34,19 @@ public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne v1
             return null;
         }
 
-        public static ExceptionData CreateExceptionData(ExceptionDataVersionOne v1ExceptionData)
```

Comment:
Yay! for reduced visibility. And I assume it's no longer `static` because somewhere down the call stack, we'll run into somebody that needs `_run`.